README.md

```diff
@@ -1,11 +1,12 @@
-# ComputeCpp SDK Readme
 
-![ComputeCpp™]
+<div align="center">
```

Comment:
Is there no way to link this new image in the old style? If you check the bottom of the document, that's where we've defined the links to keep the main body a bit more readable. At the very least, the old links should be removed from there, though I'd prefer that we add the links there instead.

Reply:
Ah that is my mistake, I totally missed the links being defined at the end of the file! I've reverted this to the old style and updated the link for it at the bottom of the document.